caffe2/serialize/inline_container.h

```diff
@@ -141,13 +145,16 @@ class CAFFE2_API PyTorchStreamWriter final {
   ~PyTorchStreamWriter();
 
  private:
+  void setup(const string& file_name);
   void valid(const char* what, const char* info = "");
   size_t current_pos_ = 0;
   std::unique_ptr<mz_zip_archive> ar_;
   std::string archive_name_;
-  std::ostream* out_;
-  std::ofstream file_stream_;
+  std::string archive_name_plus_slash_;
+  std::unique_ptr<std::ofstream> file_stream_;
```

Comment:
Why does this need to be a `unique_ptr`?  I beliefe `ofstream` can be default constructed.

Reply:
It didn't need to be a unique_ptr<>, was mostly trying to save work in the case where it's not needed. Reverted this part back.